InnerEye/ML/metrics_dict.py

```diff
@@ -662,14 +664,17 @@ def store_metrics_per_subject(self,
                     LoggingColumns.Patient.value: prediction_entry.subject_id,
                     LoggingColumns.ModelOutput.value: prediction_entry.predictions,
                     LoggingColumns.Label.value: prediction_entry.labels,
+                    LoggingColumns.Epoch.value: epoch,
                     LoggingColumns.CrossValidationSplitIndex.value: cross_validation_split_index,
                     LoggingColumns.DataSplit.value: mode.value
                 })
 
     @staticmethod
     def load_execution_mode_metrics_from_df(df: pd.DataFrame,
                                             is_classification_metrics: bool) -> Dict[ModelExecutionMode,
-                                                                                     Dict[int, ScalarMetricsDict]]:
+                                                                                     Dict[Union[
```

Comment:
you are using the very complex thing repeatedly, define a type alias?

Reply:
updated